### PR TITLE
Drop warning re: @api_key not initialized

### LINKED ISSUE #46: @api_key not initialized
I get this warning when running tests:
`airrecord/lib/airrecord/table.rb:32: warning: instance variable @api_key not initialized`

Looks like we initialize `@api_key` in `client.rb` but then try to use it in `table.rb`?

### PR DESCRIPTION
**Pull request by chrisfrank**
Fixes #46

### CODE REVIEW

File: lib/airrecord/table.rb [hunk 1]
```diff
@@ -1,13 +1,18 @@
 module Airrecord
   class Table
     class << self
-      attr_accessor :base_key, :table_name, :api_key, :associations
```
Comment (chrisfrank) [inline lib/airrecord/table.rb]: Dropped the `:associations` accessor while making this change, because v1 made it a noop.

**Comment by sirupsen** [comment]
Sweet, you want to cut 1.0.1 with this?

**Comment by chrisfrank** [comment]
Yes indeed — unless you're still seeing incorrect sorting (re: #37)?

**Comment by sirupsen** [comment]
Let's deal with that some other time :) This is fine for 1.0.1

**Comment by chrisfrank** [comment]
Great! 1.0.1 is live on rubygems.org

**Pull request merged**